Apply comments from review on the guidelines (#1331)
EuroPython/website: src/content/pages/guidelines.mdx

backup of anything you need rather than relying on the internet: while we do have dedicated ethernet
cables on the podiums, Murphy has a special nose for materials needed during live sessions.

<div class="flex gap-4 max-w-[800px]" >

>   <Image src={speakerPhoto} alt="Speaker photo" class="w-[60%] border-4 border-white rounded-lg shadow-lg" />

Reviewer: ```suggestion
  <Image src={speakerPhoto} alt="A speaker behind the podium" class="w-[60%] border-4 border-white rounded-lg shadow-lg" />
```

**Meeting Room 241** will be available as a Speaker Ready Room. There will be a monitor with an HDMI
cable if you want to check the set up for your presentation.

<div class="flex gap-4 max-w-[800px]" >

>   <Image src={posterPhoto} alt="Speaker photo" class="w-[60%] border-4 border-white rounded-lg shadow-lg" />

Reviewer: ```suggestion
  <Image src={posterPhoto} alt="People reading a poster" class="w-[60%] border-4 border-white rounded-lg shadow-lg" />
```

| South Room 223+224 | 30 |
</div>

<div class="flex gap-4 max-w-[800px]" >

>   <Image src={helperPhoto} alt="Speaker photo" class="w-[60%] border-4 border-white rounded-lg shadow-lg" />

Reviewer: Do we call those who give tutorials "tutors"?
```suggestion
  <Image src={helperPhoto} alt="A tutor helping a tutorial attendee at their laptop" class="w-[60%] border-4 border-white rounded-lg shadow-lg" />
```

give them a heads up at least a couple of days prior. Bandwidth will be limited, and many public
resources have some kind of rate limiting set up.

If you will need helpers for your session, please add them to your session in Pretalx and contact the
Programme Committee (programme@europython.eu) ahead of the conference, as we’ll need to give them
something for them to be able to attend the tutorials (in case they don't have
tutorial tickets). In the case they do have tutorial tickets, let us know

> anyways.

Reviewer: ```suggestion
anyway.
```
